src/hooks/useGardenHooks.ts

```diff
@@ -73,9 +73,9 @@ export function useGardenData() {
 
     let cancelled = false
 
-    const fetchGardenConnector = async () => {
+    const fetchGardenConnector = () => {
       try {
-        const gardenConnector: Garden = await connectGarden(organization, {
```

Comment:
The original code dont need await async because dont return Promises https://github.com/1Hive/gardens/blob/master/packages/connector/src/connect.ts